base/Package.md

```diff
@@ -1,5 +1,3 @@
-Package containing the FHIR R4 resources types
```

Comment:
We may need this line. This is the line that'll show as the package description in the search result view.